### PR TITLE
[C-1724] Add track screen skeleton

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Adds track screen skeleton, to improve perceived load when clicking a deep link track. This extra helps cause coming into the audius app, things feel extra broken having a blank screen as first experience.

TODO: collection pages

### CODE REVIEW

**Review by Kyle-Shanks — APPROVED** [review]
clean stuff, nice

**Review by sliptype — APPROVED** [review]
LETSSS GOOO

I've been meaning to do this, thank you!

I will say that we maybe don't need all this complexity in the skeleton, when I was profiling the perf of animations it turned out that having many animations on screen like this is a big perf hit. So maybe we can simplify a bit?

**Comment by dylanjeffers** [comment]
> LETSSS GOOO
> 
> I've been meaning to do this, thank you!
> 
> I will say that we maybe don't need all this complexity in the skeleton, when I was profiling the perf of animations it turned out that having many animations on screen like this is a big perf hit. So maybe we can simplify a bit?

totally agreed, how about i remove the shimmer effect for every one except maybe the image?

**Comment by dylanjeffers** [comment]
> > LETSSS GOOO
> > I've been meaning to do this, thank you!
> > I will say that we maybe don't need all this complexity in the skeleton, when I was profiling the perf of animations it turned out that having many animations on screen like this is a big perf hit. So maybe we can simplify a bit?
> 
> totally agreed, how about i remove the shimmer effect for every one except maybe the image?

The other option im thinking, is that we can somehow use a "SkeletonProvider" that manages the single animated value instance, that all these use to do a coordinated anim?

File: packages/mobile/src/screens/track-screen/TrackScreenSkeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            <StaticSkeleton style={styles.metric} />
+            <StaticSkeleton style={styles.metric} />
+            <StaticSkeleton style={styles.metric} />
+          </View>
+          <View style={styles.description}>
+            {times(random(5, 15), () => random(20, 100)).map(
+              (elementWidth: number, i) => (
+                <StaticSkeleton
+                  key={i}
+                  noShimmer
```
Comment (Kyle-Shanks) [inline packages/mobile/src/screens/track-screen/TrackScreenSkeleton.tsx:134]: can lose this `noShimmer` now right?

**Pull request merged**